source/connect/connection-options/connection-pools.txt

```diff
@@ -69,24 +128,70 @@ connection succeeds only when one the following cases occurs:
 - The driver's ability to reuse existing connections improves due to
   rate-limits on connection creation.
```

Comment:
```suggestion
```

I'm not sure what this bullet point is actually getting at, I'd suggest just removing it.